tests/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from control.config import GatewayConfig
+
+
+def pytest_addoption(parser):
+    """Sets command line options for testing."""
+    # Specify base config file for tests
+    parser.addoption("--config", action="store", default="ceph-nvmeof.conf")
+    parser.addoption("--rbd_image", action="store", default="mytestdevimage")
+
+
+@pytest.fixture(scope="session")
+def conffile(request):
+    return request.config.getoption("--config")
+
+
+@pytest.fixture(scope="session")
+def config(conffile):
```

Comment:
avoid using function name as parameter defined above. Docstring.

Reply:
Using function names as parameters is done purposefully. These functions are fixtures, which provide setup and tear down for the test functions and [are called](https://docs.pytest.org/en/6.2.x/fixture.html) by identifying them in the parameter list.

The missing docstring was not done purposefully, I will add that.

Reply:
Ah got it. Thank you!